### ISSUE TITLE
Column not found: 1054 Unknown column 'freeform_submissions.num' in 'order clause'

### ISSUE THREAD

**Issue by 1stevengrant** [solspace/craft-freeform#299]
Hey folks,
I have a Freeform form that we are conditionally relating back to a Craft user account like so

```twig
{{ form.renderTag({
					returnUrl: entry.url ~ '?applied',
					dynamicNotification: {
						recipients: [entry.applyEmail],
						template: "employer.html"
					},
					relations: {
						applications: currentUser ? currentUser.id : null
					}
				}) }}
```

The form itself works great and submissions are appearing as we'd expect.

When I visit the user profile in the control panel, our applications field correctly shows the form submission relation. However, when I try to double-click the submission to preview, I run into an error.
```log
{"error":"SQLSTATE[42S22]: Column not found: 1054 Unknown column 'freeform_submissions.num' in 'order clause'\nThe SQL being executed was: SELECT `elements`.`id`, `elements`.`fieldLayoutId`, `elements`.`uid`, `elements`.`enabled`, `elements`.`archived`, `elements`.`dateCreated`, `elements`.`dateUpdated`, `elements_sites`.`id` AS `siteSettingsId`, `elements_sites`.`slug`, `elements_sites`.`siteId`, `elements_sites`.`uri`, `elements_sites`.`enabled` AS `enabledForSite`, `elements`.`canonicalId`, `elements`.`dateLastMerged`, freeform_submissions.`formId`, freeform_submissions.`userId`, freeform_submissions.`statusId`, freeform_submissions.`incrementalId`, freeform_submissions.`token`, freeform_submissions.`isSpam`, freeform_submissions.`ip`, freeform_submissions.`field_14`, freeform_submissions.`field_3`, freeform_submissions.`field_16`, freeform_submissions.`field_18`, freeform_submissions.`field_13`, freeform_submissions.`field_15`, freeform_submissions.`field_10`, freeform_submissions.`field_1`, freeform_submissions.`field_2`, freeform_submissions.`field_4`, freeform_submissions.`field_5`, freeform_submissions.`field_6`, freeform_submissions.`field_7`, freeform_submissions.`field_9`, freeform_submissions.`field_11`, freeform_submissions.`field_19`, freeform_submissions.`field_20`, freeform_submissions.`field_8`, freeform_submissions.`field_12`, freeform_submissions.`field_17`, `content`.`id` AS `contentId`, `content`.`title`, `content`.`field_apiApplyUrl_jlvjyxcj`, `content`.`field_apiFeedName_wojrjtst`, `content`.`field_apiLocation_kuuafntb`, `content`.`field_apiSpecialty_keqxkjdu`, `content`.`field_apiType_sruiofya`, `content`.`field_applyEmail_dtpcvbek`, `content`.`field_applyUrl_hmuzfmpb`, `content`.`field_body_qdnvsxcv`, `content`.`field_city_feahijtt`, `content`.`field_club_knjkmkzz`, `content`.`field_country_rdyaljgw`, `content`.`field_dateClosing_ajstcqwh`, `content`.`field_directListing_syldbslo`, `content`.`field_embedCode_ezrfnjkk`, `content`.`field_hasSavedProfile_pomldukp`, `content`.`field_isCandidate_rxjtzjry`, `content`.`field_isEmployer_xnnpbwuf`, `content`.`field_isSportsworkPartner_yoqnuyer`, `content`.`field_jobDescription`, `content`.`field_klaviyoProfileId_fqbumomu`, `content`.`field_maximumSalary_qketcyvk`, `content`.`field_minimumSalary_nckuuiwc`, `content`.`field_organisationBio_fuvazjkf`, `content`.`field_organisationName_rdpygnjn`, `content`.`field_otherSport_fbscgtdr`, `content`.`field_phone_mnxcikbr`, `content`.`field_postcode_ginrvass`, `content`.`field_pushToSocialMedia`, `content`.`field_quote_lwnhdqqm`, `content`.`field_salaryCurrency_bkvaghrb`, `content`.`field_stateRegion_uvbcpngl`, `content`.`field_streetAddress_bqaaehny`, `content`.`field_subTitle_lyeenoum`, `elements`.`revisionId`, `revisions`.`creatorId` AS `revisionCreatorId`, `revisions`.`num` AS `revisionNum`, `revisions`.`notes` AS `revisionNotes`\nFROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `elementsSitesId`, `content`.`id` AS `contentId`\nFROM `elements` `elements`\nINNER JOIN `freeform_submissions` `freeform_submissions` ON `freeform_submissions`.`id` = `elements`.`id`\nINNER JOIN `freeform_statuses` `sub_freeform_statuses` ON sub_freeform_statuses.`id` = freeform_submissions.`statusId`\nINNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id`\nINNER JOIN `content` `content` ON `content`.`elementId` = `elements`.`id`\nINNER JOIN `revisions` `revisions` ON `revisions`.`id` = `elements`.`revisionId`\nINNER JOIN `freeform_forms` `freeform_forms` ON freeform_forms.`id` = freeform_submissions.`formId`\nINNER JOIN `freeform_statuses` `freeform_statuses` ON freeform_statuses.`id` = freeform_submissions.`statusId`\nWHERE (freeform_submissions.`isSpam`=FALSE) AND (`elements`.`archived`=FALSE) AND (`elements`.`dateDeleted` IS NULL) AND (`elements`.`dateCreated`='2022-03-11 10:55:14') AND (`elements`.`draftId` IS NULL) AND (`elements`.`canonicalId`=16063)\nORDER BY freeform_submissions.`num` DESC\nLIMIT 1) `subquery`\nINNER JOIN `freeform_submissions` `freeform_submissions` ON `freeform_submissions`.`id` = `subquery`.`elementsId`\nINNER JOIN `elements` `elements` ON `elements`.`id` = `subquery`.`elementsId`\nINNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`elementsSitesId`\nINNER JOIN `content` `content` ON `content`.`id` = `subquery`.`contentId`\nINNER JOIN `revisions` `revisions` ON `revisions`.`id` = `elements`.`revisionId`\nINNER JOIN `freeform_forms` `freeform_forms` ON freeform_forms.`id` = freeform_submissions.`formId`\nINNER JOIN `freeform_statuses` `freeform_statuses` ON freeform_statuses.`id` = freeform_submissions.`statusId`\nORDER BY freeform_submissions.`num` DESC","exception":"yii\\db\\Exception","file":"/home/vendor/yiisoft/yii2/db/Schema.php","line":676,"trace":[{"file":"/home/vendor/yiisoft/yii2/db/Command.php","line":1307,"function":"convertException","class":"yii\\db\\Schema","type":"->"},{"file":"/home/vendor/yiisoft/yii2/db/Command.php","line":1168,"function":"internalExecute","class":"yii\\db\\Command","type":"->"},{"file":"/home/vendor/yiisoft/yii2/db/Command.php","line":424,"function":"queryInternal","class":"yii\\db\\Command","type":"->"},{"file":"/home/vendor/yiisoft/yii2/db/Query.php","line":287,"function":"queryOne","class":"yii\\db\\Command","type":"->"},{"file":"/home/vendor/craftcms/cms/src/db/Query.php","line":167,"function":"one","class":"yii\\db\\Query","type":"->"},{"file":"/home/vendor/craftcms/cms/src/elements/db/ElementQuery.php","line":1639,"function":"one","class":"craft\\db\\Query","type":"->"},{"file":"/home/vendor/craftcms/cms/src/base/Element.php","line":3747,"function":"one","class":"craft\\elements\\db\\ElementQuery","type":"->"},{"file":"/home/vendor/craftcms/cms/src/base/Element.php","line":4077,"function":"getCurrentRevision","class":"craft\\base\\Element","type":"->"},{"file":"/home/vendor/craftcms/cms/src/helpers/Cp.php","line":902,"function":"craft\\base\\{closure}","class":"craft\\base\\Element","type":"->"},{"file":"/home/vendor/craftcms/cms/src/base/Element.php","line":3975,"function":"metadataHtml","class":"craft\\helpers\\Cp","type":"::"},{"file":"/home/vendor/craftcms/cms/src/controllers/ElementsController.php","line":478,"function":"getSidebarHtml","class":"craft\\base\\Element","type":"->"},{"file":"/home/vendor/craftcms/cms/src/web/View.php","line":1529,"function":"craft\\controllers\\{closure}","class":"craft\\controllers\\ElementsController","type":"->"},{"file":"/home/vendor/craftcms/cms/src/controllers/ElementsController.php","line":477,"function":"namespaceInputs","class":"craft\\web\\View","type":"->"},{"file":"/home/vendor/craftcms/cms/src/controllers/ElementsController.php","line":139,"function":"_getEditorHtmlResponse","class":"craft\\controllers\\ElementsController","type":"->"},{"function":"actionGetEditorHtml","class":"craft\\controllers\\ElementsController","type":"->"},{"file":"/home/vendor/yiisoft/yii2/base/InlineAction.php","line":57,"function":"call_user_func_array"},{"file":"/home/vendor/yiisoft/yii2/base/Controller.php","line":178,"function":"runWithParams","class":"yii\\base\\InlineAction","type":"->"},{"file":"/home/vendor/yiisoft/yii2/base/Module.php","line":552,"function":"runAction","class":"yii\\base\\Controller","type":"->"},{"file":"/home/vendor/craftcms/cms/src/web/Application.php","line":293,"function":"runAction","class":"yii\\base\\Module","type":"->"},{"file":"/home/vendor/craftcms/cms/src/web/Application.php","line":602,"function":"runAction","class":"craft\\web\\Application","type":"->"},{"file":"/home/vendor/craftcms/cms/src/web/Application.php","line":272,"function":"_processActionRequest","class":"craft\\web\\Application","type":"->"},{"file":"/home/vendor/yiisoft/yii2/base/Application.php","line":384,"function":"handleRequest","class":"craft\\web\\Application","type":"->"},{"file":"/home/web/index.php","line":26,"function":"run","class":"yii\\base\\Application","type":"->"},{"file":"/home/.composer/vendor/laravel/valet/server.php","line":234,"function":"require"}]}

```
Is this a known bug?

Thanks,
Steven

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.7.36
- Freeform Version: 3.13.4
- Freeform Edition: Pro

**Comment by kjmartens** [comment]
Hi @1stevengrant. I'm very sorry for the long delay on this one.

I can duplicate this issue on my end as well. However, I have a hunch that it might be something we can't resolve within the slide-out modal because Freeform submissions are "half" elements, where some common data is stored as Craft content while the field data is stored within Freeform's own tables. I noticed that in Craft 4, this doesn't happen (the slideout doesn't happen at all, therefore no errors are triggered).

At any rate, I will make a note about this and see if it's something that we can get working. 🙂

**Comment by 1stevengrant** [comment]
No worries sir. Since moved on from that project but thought it good to highlight.

**Comment by kjmartens** [comment]
This how now been resolved in Freeform 3.13.18 and 4.0.1. 🙂
